ui/helpers/constants/settings.js

```diff
@@ -419,13 +419,6 @@ const SETTINGS_CONSTANTS = [
     route: `${ABOUT_US_ROUTE}#beta-terms`,
     iconName: IconName.Info,
   },
-  {
-    tabMessage: (t) => t('advanced'),
```

Comment:
All settings options should be listed here for the purpose of supporting the settings "search" feature. Rather than removing this entry, we should update it to match the renamed button

Reply:
Additionally, the "Restore user data" entry below this is no longer valid, as that was already removed in #26325. We missed this entry in that PR.

Reply:
Added with comments in [3802a74](https://github.com/MetaMask/metamask-extension/pull/26322/commits/3802a742d8df1d510983c747b636a11e46777f8d)